app/components/dashboard/DcChart.vue

```
@@ -2787,124 +2694,56 @@ const mm = {
-              break;
-            }
-            if (ymd === mm.opt.assets.chartDateLineYmdMsg[20][0]) {
-              s_suf = mm.opt.assets.chartDateLineYmdMsg[20][1];
-              break;
-            }
-            break;
-        }
-      }
-
+      // unit = ' ' + unit;
+      let isBar = isNaN(d.value);
```

ESLint

[error] 'isBar' is never reassigned. Use 'const' instead.

(prefer-const)